apis/validator/inclusion_list.yaml

```diff
@@ -0,0 +1,62 @@
+get:
+  tags:
+    - EIP-7805
+  operationId: "produceInclusionList"
+  summary: "Produce an inclusion list"
+  description: Requests the beacon node to produce an inclusion list.
+  parameters:
+    - name: slot
+      in: query
+      required: true
+      description: "The slot for which an inclusion list should be created."
+      schema:
+        $ref: ../../beacon-node-oapi.yaml#/components/schemas/Uint64
+  responses:
+    "200":
+      description: Success response
+      content:
+        application/json:
+          schema:
+            title: ProduceInclusionListResponse
+            type: object
+            required: [data]
+            properties:
+              data:
+                $ref: "../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.Transactions"
```

Comment:
is this meant to return `EIP7805.InclusionList`? maybe I'm reading things wrong but this seems to expect the transactions of the IL in the data property whereas description says this will request BN to produce an IL.

Reply:
depends how you define what a inclusion list is, technically it's just a list of transactions which is the return type of this api and also the engine-api

the other return type could be `InclusionList` (ie. wrap it in container) but this would return useless and even confusing data, eg. `validator_index` isn't even set by beacon node, so beacon node would just set a random value (maybe `0`) and then the validator client is responsible for overriding it

we can think about if we wanna wrap this in a container, and maybe even make the api fork-aware by adding version metadata, this might be better for future forward compatibility

Reply:
> is this meant to return `EIP7805.InclusionList`? maybe I'm reading things wrong but this seems to expect the transactions of the IL in the data property whereas description says this will request BN to produce an IL.

Yes. As @nflaig mentioned, IL can mean 
1) A list of IL transactions. It is used here, [execution api spec](https://github.com/ethereum/execution-apis/blob/e70099c8408affe43bf227d76ce9c33ec9ff6882/src/engine/experimental/eip7805.md#engine_getinclusionlistv1) and anything as far as EL is concerned 
2) IL container as defined in consensus specs. Basically 1 + any other useful consensus info


Here we only return IL transactions because any additional consensus info (slot, validator index, ILC root) are all available in VC. No need to grab them from BN